learnrl/agents/standard/__init__.py

```diff
@@ -27,9 +27,9 @@ class StandardAgent(Agent):
         action_space: |gym.Space|
             The action_space of the environement that the agent will act on
         control: :class:`~learnrl.control.Control`
-            Control object to define policy from :attr:`action_value` *(default is 0.1-Greedy)*
+            Control object to define policy from :attr:`learnrl.agents.standard.StandardAgent.action_values` *(default is 0.1-Greedy)*
```

Comment:
here you should add the ~ juste after ` like this : `~learnrl.age...` as such, it will only display action_values and not learnrl.agents.standard.StandardAgent.action_values in the documentation !

Reply:
Also, to check the documentation after modifications, try to build your local documentation:
`pip install -r docs\requirements.txt` then `docs\make.bat html`

Reply:
ohhk got it.

Reply:
while installing requirements from requirements.txt file it ended with 404 not found for https://pypi.org/simple/requirements-txt/

Reply:
I think we need to update requirements.txt as well

Reply:
Strange let me check that

Reply:
No problem on my side, can you file an issue for that ? We will deal with that on an other pull request

Reply:
> 
> 
> while installing requirements from requirements.txt file it ended with 404 not found for https://pypi.org/simple/requirements-txt/

Maybe you forgot the '-r'

Reply:
yeah got it.